client/lib/signup/test/progress-store-test.js

```diff
@@ -31,17 +33,19 @@ describe( 'SignupProgressStore', function() {
 		assert.equal( SignupProgressStore.get()[ 0 ].stepName, 'site-selection' );
 	} );
 
-	it( 'should add a timestamp to each step', function( done ) {
-		var previousTimestamp;
+	it( 'should add a timestamp to each step', function() {
+		var previousTimestamp,
+			clock = sinon.useFakeTimers();
 
 		SignupActions.saveSignupStep( { stepName: 'site-selection' } );
+		clock.tick( 100 );
 		previousTimestamp = SignupProgressStore.get()[ 0 ].lastUpdated;
 
-		setTimeout( function() {
-			SignupActions.saveSignupStep( { stepName: 'site-selection' } );
-			assert( SignupProgressStore.get()[ 0 ].lastUpdated > previousTimestamp );
-			done();
-		}, 5 );
+		SignupActions.saveSignupStep( { stepName: 'site-selection' } );
+		clock.tick( 100 );
+		assert( SignupProgressStore.get()[ 0 ].lastUpdated > previousTimestamp );
+
+		clock.restore();
```

Comment:
With longer test suites, it's safer to move stubbing and restoring to before/after or beforeEach/afterEach blocks.